ts/IterableInboxMessageCell.tsx

```diff
@@ -1,25 +1,23 @@
 'use strict'
 
-import React from 'react'
+import React, { useRef } from 'react'
```

Comment:
Just a nit pick.
Not sure whats the best practice. Should we write all the modules within same package in curly braces? some outside and some inside looks unformatted.

Reply:
This is the convention. If I include React within the braces, I get an error that React can only be imported as a default import.